lib/Common/DataStructures/List.h

```diff
@@ -360,6 +360,7 @@ namespace JsUtil
         void SetItem(int index, const T& item)
         {
             EnsureArray(index + 1);
+            // TODO: (SWB)(leish) find a way to force user defined copy constructor
```

Comment:
Got you. Might the answer be impossible (language deficit? operator= and copy constructor are different no automatic behavior)? If so this comment is not useful.

---
In reply to: [92256073](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92256073) [](ancestors = 92256073,92253745)